conda_build/conda_interface.py

```diff
@@ -87,6 +87,14 @@
     PackageRecord,
     addendum="Use `conda.models.records.PackageRecord` instead.",
 )
+deprecated.constant("24.1.0", "24.3.0", "Dist", Dist)
+deprecated.constant("24.1.0", "24.3.0", "display_actions", display_actions)
+deprecated.constant("24.1.0", "24.3.0", "execute_actions", execute_actions)
+deprecated.constant("24.1.0", "24.3.0", "execute_plan", execute_plan)
+deprecated.constant("24.1.0", "24.3.0", "get_index", get_index)
+deprecated.constant("24.1.0", "24.3.0", "install_actions", install_actions)
+deprecated.constant("24.1.0", "24.3.0", "linked", linked)
+deprecated.constant("24.1.0", "24.3.0", "linked_data", linked_data)
```

Comment:
`deprecated.constant` uses the module level `__getattr__` to detect usage of deprecated variables which only occurs if the variable doesn't exist, we need to change all of these imports into:

```python
from conda.models.dist import Dist as _Dist
```

```suggestion
deprecated.constant("24.1.0", "24.3.0", "Dist", _Dist)
deprecated.constant("24.1.0", "24.3.0", "display_actions", _display_actions)
deprecated.constant("24.1.0", "24.3.0", "execute_actions", _execute_actions)
deprecated.constant("24.1.0", "24.3.0", "execute_plan", _execute_plan)
deprecated.constant("24.1.0", "24.3.0", "get_index", _get_index)
deprecated.constant("24.1.0", "24.3.0", "install_actions", _install_actions)
deprecated.constant("24.1.0", "24.3.0", "linked", _linked)
deprecated.constant("24.1.0", "24.3.0", "linked_data", _linked_data)
```

Reply:
Good call, thanks.